public/app/features/explore/RichHistory/RichHistoryCard.test.tsx

```diff
@@ -304,117 +298,6 @@ describe('RichHistoryCard', () => {
     });
   });
 
-  describe('run queries', () => {
-    it('should be disabled if at least one query datasource is missing when using mixed', async () => {
```

Comment:
This test was not accurate and was removed. A test that actually works will be developed for https://github.com/grafana/grafana/issues/88001